src/resources/timeSeries/timeSeriesApi.ts

```diff
@@ -149,6 +158,24 @@ export class TimeSeriesAPI extends BaseResourceAPI<
     return super.deleteEndpoint(ids);
   };
 
+  /**
+   * [Synthetic Query](https://docs.cognite.com/api/v1/#operation/querySyntheticTimeseries)
+   *
+   * ```js
+   * await client.timeseries.syntheticQuery([
+   *   {
+   *     expression: "24 * TS{externalId='production/hour', aggregate='average', granularity='1d'}",
+   *     start: 0,
+   *     end: 0,
```

Comment:
```suggestion
   *     end: 0,
```
both start and end can't be zero.
maybe use relative time?